vSync/vs_client.lua

```diff
@@ -91,3 +91,122 @@ RegisterNetEvent('vSync:notify')
 AddEventHandler('vSync:notify', function(message, blink)
     ShowNotification(message, blink)
 end)
+
+--[[ UNIVERSAL MENU HOOKING STUFF ]]--
```

Comment:
maybe add a link to where people can get/find info about this universal menu? I don't mind a link there just so people know what it's for 😉